src/config.js

```diff
@@ -159,8 +159,7 @@ export const inputKeyToStateKeyMap = {
 
 export const PACKAGE_FSSPEC = 'fsspec';
 
-export const KEDRO_VIZ_DOCS_URL =
-  'https://docs.kedro.org/projects/kedro-viz/';
+export const KEDRO_VIZ_DOCS_URL = 'https://docs.kedro.org/projects/kedro-viz/';
 export const KEDRO_VIZ_PUBLISH_DOCS_URL = `${KEDRO_VIZ_DOCS_URL}share_kedro_viz.html`;
```

Comment:
not related to this PR but these link with `.html` wont work with the new mkdocs anymore after the release. I'll make a separate PR to update them